### ISSUE TITLE
Add Contributors.md

### ISSUE THREAD

**Issue by johngruen** [mozilla-services/screenshots#2380]
Could be two bugs, but we should add all things contrib related!

**Comment by ianb** [comment]
Note we already have contributing.json

**Comment by johngruen** [comment]
@ianb my work here is done

**Comment by pdehaan** [comment]
```sh
$ git shortlog -s | cut -c8- | sort -f > CONTRIBUTORS.md

$ cat CONTRIBUTORS.md

Abel 'Akronix' Serrano Juste
Ben Ennis
Bizarro @clouserw (minimal permissions)
Bram Pitoyo
Chuck Harmston
Cory Price
Daniel Thorn
Danny Coates
Donovan Preston
Erik Vold
greenkeeperio-bot
Ian Bicking
Jared Hirsch
John Gruen
Kit Cambridge
Mark Banner
Michael R. Bernstein
niharikak101
Peter deHaan
Sadika Sumanapala
Wil Clouser
YFdyh000
```

**Comment by ianb** [comment]
Is there any way to map shortlog names to GitHub usernames?  It would be nice to link to their accounts.  (I unfortunately don't see a way)

We could in the future hook this into the release notes scripting.

**Comment by pdehaan** [comment]
> Is there any way to map shortlog names to GitHub usernames?

Not that I can find. I think some answers on The 'Net are suggesting piping the emails through the GitHub API to try and convert emails to usernames (but may not always work if a user doesn't have a public email on GitHub).

Possibly use https://api.github.com/repos/mozilla-services/pageshot/contributors, which will give us the GitHub usernames, but not the user's name/email. ¯\\\_(ツ)\_/¯

**Comment by 6a68** [comment]
closed by #2492
